app/script/conversation/ConversationMapper.js

```diff
@@ -217,65 +217,67 @@ z.conversation.ConversationMapper = class ConversationMapper {
    * @param {Array} remote - Backend payload
    * @returns {Array} Merged conversation data
    */
-  merge_conversations(local, remote) {
+  mergeConversation(local, remote) {
     return remote.map((remote_conversation, index) => {
-      const {access, access_role, id, creator, members, name, team, type} = remote_conversation;
-      let local_conversation = local.filter(conversation => conversation).find(conversation => conversation.id === id);
-
-      if (!local_conversation) {
-        local_conversation = {id};
-      }
-
-      local_conversation.accessModes = access;
-      local_conversation.accessRole = access_role;
-      local_conversation.creator = creator;
-      local_conversation.name = name;
-      local_conversation.status = members.self.status;
-      local_conversation.team_id = team;
-      local_conversation.type = type;
+      const {access, access_role, id, creator, members, message_timer, name, team, type} = remote_conversation;
+      const localConversation = local
+        .filter(conversation => conversation)
+        .find(conversation => conversation.id === id) || {id};
```

Comment:
@atomrc Do you think this is good to read or can we find another solution that prettier is happy with 🙈 

Maybe we can change this to:
`const localConversation = local.filter(conversation => conversation).find(conversation => conversation.id === id);`

And do:
`const mergedConversation = Object.assign({}, localConversation || {id}, updates);`

Wdyt?